charts/node-analyzer/README.md

```diff
@@ -180,7 +185,18 @@ $ helm install --namespace nodeanalyzer node-analyzer -f values.yaml sysdig/node
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
-## Chart Components
+## Node Analyzer
+
+The Node Analyzer is deployed by default unless you set the value `nodeAnalyzer.deploy` to `false`.
+
+The Node Analyzer daemonset contains three containers, each providing a specific functionality. This daemonset replaces
+the (deprecated) Node Image Analyzer daemonset.
+
+See
+the [Node Analyzer installation documentation](https://docs.sysdig.com/en/node-analyzer--multi-feature-installation.html)
+for details about installation, and
+[Running Node Analyzer Behind a Proxy](https://docs.sysdig.com/en/node-analyzer--multi-feature-installation.html#UUID-35c14c46-b327-c2a8-ed9c-82a2af995218_section-idm51621039128136)
```

Comment:
Not really an issue but both the links point to the same page starting at the top although both seem to have different links.

Reply:
yup, these look outdated. fixed this and a few other instances of outdated anchors